src/component/calculator.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+/* eslint-disable */
+
+class Calculator extends Component {
+  render() {
+    return (
```

Comment:
- [ ] Nice work so far 👍🏿  in your project. But as per project requirement, you are expected to use functional components instead of the class components you are currently using